typing/cmt_format.mli

```diff
@@ -91,8 +91,7 @@ val save_cmt :
   binary_annots ->
   string option ->  (* source file *)
   Env.t -> (* initial env *)
-  Types.signature option -> (* if a .cmi was generated,
-                               the signature saved there *)
+  Cmi_format.cmi_infos option -> (* if a .cmi was generated *)
```

Comment:
The doc comment of the function should probably be changed to say `cmi` instead of `sg`. Also, this is not related to this patch but the order of argument names looks wrong